tests/quantization/ggml/test_ggml.py

```diff
@@ -825,8 +825,8 @@ def test_nemotron_q6_k(self):
             gguf_file=self.q6_k_nemotron_model_id,
             dtype=torch.float16,
         )
-
-        tokenizer = AutoTokenizer.from_pretrained(self.nemotron_model_id, gguf_file=self.q6_k_nemotron_model_id)
+        # use the original tokenizer from nvidia to avoid long load times
+        tokenizer = AutoTokenizer.from_pretrained("nvidia/Nemotron-Mini-4B-Instruct")
```

Comment:
only issue is this one is not a GGUF! So we are not testing the load of the tokenizer!